src/requests/request.ts

```diff
@@ -19,7 +19,7 @@ export default class PdfGenerationRequest {
         this.url = args.url;
         this.fileName = args.fileName ?? `${crypto.randomUUID()}.pdf`;
         this.localFileName = `${crypto.randomUUID()}.pdf`;
-        this.path = `/${DEFAULT_TEMP_PATH}`;
+        this.path = args.path ? `${args.path}/` : `/${DEFAULT_TEMP_PATH}`;
```

Comment:
This should probably be `args.path ?? '/${DEFAULT_TEMP_PATH}'`. If they are passing in the path they should have full control, and we shouldn't be modifying that at all. Let the outside caller handle that